web/src/components/eventdetail/commonevent.js

```diff
@@ -0,0 +1,58 @@
+import {
+  Grid,
+  Hidden,
+  Typography,
+} from '@material-ui/core';
+import PropTypes from 'prop-types';
+import React, {Component} from 'react';
+import Tags from './tags';
+import Ticket from './ticket';
+
+class CommonEvent extends Component {
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    const monthNames = ['January', 'February', 'March', 'April', 'May',
+      'June', 'July', 'August', 'September', 'October', 'November', 'December'];
```

Comment:
kinda ugly. Another way to do this is something like:

const date = new Date(2009, 10, 10);  // 2009-11-10
const month = date.toLocaleString('default', { month: 'long' });

https://stackoverflow.com/questions/1643320/get-month-name-from-date